src/main.c

```diff
@@ -580,6 +581,11 @@ int main(int argc, char* argv[]) {
         ret = JQ_OK;
         goto out;
       }
+      if (isoption(argv[i], 'V', "config", &short_opts)) {
```

Comment:
Note that `-V` is for `--version`. I don't think we will normally use this option, so we don't need a short option.

Reply:
Oops, yes.